osgi.specs/docbook/core/006/framework.resource.xml

```diff
@@ -346,7 +346,11 @@
       that must match the <code>effective</code> directive.</para>
 
       <para>For the OSGi Framework, the name <code>resolve</code> is reserved,
-      this is also the default. The syntax is therefore:</para>
+      this is also the default. Requirements and Capabilities may specify another
+      value, such as <code>active</code> for requirements that must be present for
+      a bundle to become active executable (that is for example services it needs to consume)
+      but for the OSGi Framework the only value that makes the Requirement or Capability effective is the <code>resolve</code> value.
+      The syntax is therefore:</para>
```

Comment:
This goes too far and implies the frame processes the requirement or capability when it is set to `active` before a bundle is allowed to become active.  Also, the term "active executable" is not used anywhere else in the specification (that I remember).  I'm not sure what it means.

I suggest we use what I recommended in the issue:

> For the OSGi Framework, the name resolve is reserved, this is also the default. Requirements and Capabilities may specify another value, such as active, but for the OSGi Framework the only value that makes the Requirement or Capability effective is the resolve value. The syntax is therefore: